spec/features/user_registers_spec.rb

```diff
@@ -0,0 +1,81 @@
+require 'spec_helper'
+
+feature "user registers", {js: true, vcr: true} do
+  background do
+    visit register_path
+  end  
+  
+  scenario "with valid user info and valid card" do
+    fill_valid_user
+    fill_valid_card    
+    click_button "Sign Up"
+    expect(page).to have_content("Profile was successfully created. Please sign in.")
```

Comment:
Be careful coupling your implementation too tightly to your specs. Generally I recommend just checking for a flash, if that is available. You don't want a failing spec just because you change verbiage.